dockerfiles/theia/build.sh

```diff
@@ -22,9 +22,9 @@ tar rf ${DIR}/asset-che-theia.tar .git-che-theia-sha1
 gzip -f ${DIR}/asset-che-theia.tar
 
 # Download plugins
-THEIA_YEOMAN_PLUGIN="${DIR}/asset-untagged-c11870b25a17d20bb7a7-theia_yeoman_plugin.theia"
+THEIA_YEOMAN_PLUGIN="${DIR}/asset-untagged-8a7963262e021dab8ae0-theia_yeoman_plugin.theia"
```

Comment:
if we make the filename generic (not containing the sha) then my downstream implementation would be simpler/more generic.

Reply:
Done

Reply:
needs the "asset-untagged-" prefix for the che-theia and crw-theia build to find it via wildcard search for `asset-*` files. I was just asking that you remove the SHA part.

Reply:
Changed again